copter/source/docs/traditional-helicopter-rsc-setup.rst

```diff
@@ -4,16 +4,32 @@
 Rotor Speed Control Setup
 =========================
 
-Rotor speed controller set up is described in video 3
-for electric powered helicopters and video 4 for piston 
-powered helicopters on the following page.
+The Rotor Speed Control (RSC) uses the throttle output to the engine to control rotor speed. This can be done in a few different ways either through an open loop control called a throttle curve, a set point signal to an external governor or electronic speed controller, or a closed loop internal governor. The RSC also controls the throttle for ground idle, rotor run up and shut down. This wiki covers the set up of each of the parameters in the RSC.
 
-:ref:`traditional-helicopter-configuration`
+RSC mode
+========
 
+First, set the RSC Mode parameter (:ref:`H_RSC_MODE <H_RSC_MODE>`). The RSC modes are listed below with a short description. 
+
+RC Passthrough - this mode passes through the RC channel input on which the motor interlock is assigned. 
```

Comment:
a bullet list would look better. for the options...  each bullet line starts with # and a space.....dont need to wory about this, I will fix once merged so you dont have to mess with it...